source/ID/concordium-did.rst

```diff
@@ -0,0 +1,661 @@
+      },
+      { "id": "did:ccd:sci:IND:SUBIND#entrypoint-revocationKey",
+        "name": "revocationKey"
+      }
+    ]
+  }
+
+Where ``IND`` and ``SUBIND`` are the contract index and subindex.
+``NET`` and ``ADDR`` correspond to the network and to the owner's account address.
+
+
+.. _concordium-did-pkc:
+
+Public Key Cryptography DID
+---------------------------
+
+The goal of the Public Key Cryptography DID is to represent a public key and the corresponding signature verification method.
+
+The Public Key Cryptography DID Document MUST contain the following data:
+
+- ``id`` - the DID of the public key.
+- ``verificationMethod`` - specifies a `DID verification method <did-vefication-method_>`_ for verifying a signature corresponding to the public key.
+- ``authentication`` - authentication method for the key.
+
+.. code-block:: json
+
+  {
+    "id": "did:ccd:pkc:XX",
+    "verificationMethod": [
+      {
+        "id": "did:ccd:pkc:XX#key-0",
+        "type": "Ed25519VerificationKey2020",
+        "controller": "did:ccd:NET:pkc:PK",
+        "publicKeyMultibase": "fXX"
+      }
+    ],
+    "authentication": [
+      {
+        "did:ccd:pkc:XX#key-0"
+      }
+    ]
+  }
+
+Identity Provider DID
+---------------------
+
+The goal of the Identity Provider DID is identify a Concodrium identity provider (IDP).
+An identity provider is an organization, approved by Concordium, that performs off-chain identification of users.
+IDPs are used in the account creation process to issue an identity.
+IDP DIDs can represent an issuer of a verifiable credential.
+
+The Identity Provider DID Document MUST contain the following data:
+- ``id`` - the DID of the IDP.
+- ``name`` - the IDP name.
+- ``url`` - A link to more information about the IDP.
+- ``description`` - A free form description the IDP.
+- ``verificationMethod`` - specifies a `DID verification method <did-vefication-method_>`_ for verifying a signature corresponding to the public key.
+
+.. code-block:: json
+
+  {
+    "id": "did:ccd:testnet:idp:3",
+    "name": "Digital Trust Solutions TestNet",
+    "url": "https://www.digitaltrustsolutions.nl",
+    "description": "Identity verified by Digital Trust Solutions on behalf of Concordium",
+    "verificationMethod": [
+      {
+        "id": "did:ccd:testnet:idp:3#cdi-key",
+        "type": "Ed25519VerificationKey2020",
+        "controller": "did:ccd:NET:pkc:PK",
+        "publicKeyMultibase": "fXX"
+      }
+    ]
+  }
+
+
+Concordium DID Operations
+=========================
+
+Concordium DIDs are managed on the Concordium blockchain.
+
+Create
+------
+
+Account DID
+^^^^^^^^^^^
+
+An account DID can be created by `opening an account <concordium-accounts_>`_ on the ``NET`` blockchain.
+The resulting DID is ``did:ccd:NET:acc:ADDR`` where ``ADDR`` is the base58 encoded account address.
+
+Concordium Credential DID
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A Concordium Credential DID is created as part of the account opening process.
```

Comment:
Or as part of adding acredential onto an existing account.

Reply:
Fixed.